com.saleae.Logic.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "com.saleae.Logic",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "logic-wrapper",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--device=all",
+        "--env=LD_LIBRARY_PATH=/app/extra/lib",
+        "--env=QT_ACCESSIBILITY=0",
+        "--env=NO_AT_BRIDGE=1"
+    ],
+    "build-options": {
+        "build-args": [
+            "--with-appdir"
```

Comment:
What's this for?

Reply:
More importantly, it breaks the sandboxed build https://flathub.org/builds/#/builders/2/builds/3703